.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,150 @@
+name: 🐛 Bug Report
+description: Help us improve this repository by filing a detailed bug report.
+body:
+  - type: input
+    id: nodeVersion
+    attributes:
+      label: Node Version
+      description: The version of Node.js you are using (only LTS versions are supported).
+      placeholder: 20.0.0
+    validations:
+      required: true
+
+  - type: input
+    id: pmVersion
+    attributes:
+      label: NPM/Yarn/PNPM Version
+      description: The package manager and version you are using.
+      placeholder: NPM 10.0.0
+    validations:
+      required: true
+
+  - type: input
+    id: strapiVersion
+    attributes:
+      label: Strapi Version
+      description: The version of Strapi you are using ("Latest" is not a valid response).
+      placeholder: 5.7.0
+    validations:
+      required: true
+
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: Select the operating system where you are experiencing the issue.
+      multiple: false
+      options:
+        - Strapi Cloud
+        - MacOS
+        - Linux (Debian/Ubuntu)
+        - Linux (RedHat/CentOS)
+        - Linux (Other)
+        - Windows 10
+        - Windows 11
```

Comment:
Why are we asking the version for Windows but not for the other OS? Should we just ask "Windows"?

Reply:
It kind of makes sense just because trying to provide a list for the others would be unmanageable and windows only has 2 😆

But also, maybe make it a multiselect?  For things like strapi cloud + macOs,  or docker + linux. Or split it into OS + OS Version fields?

For now I think it's fine and we can tweak it later though